core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -320,9 +320,10 @@ object TopicCommand extends Logging {
         val allConfigs = adminClient.describeConfigs(topics.map(new ConfigResource(Type.TOPIC, _)).asJavaCollection).values()
         val liveBrokers = adminClient.describeCluster().nodes().get().asScala.map(_.id())
         val topicDescriptions = adminClient.describeTopics(topics.asJavaCollection).all().get().values().asScala
+          .toSeq.sortBy(td => td.name())
         val describeOptions = new DescribeOptions(opts, liveBrokers.toSet)
         val topicPartitions = topicDescriptions
-          .flatMap(td => td.partitions.iterator().asScala.map(p => new TopicPartition(td.name(), p.partition())))
+          .flatMap(td => td.partitions.iterator().asScala.map(p => new TopicPartition(td.name(), p.partition())).toSeq)
```

Comment:
why `toSeq` is required?

Reply:
It doesn't need it, i have deleted it. :)